### PR TITLE
Doc: Add scripts for manual registration + improve manual reset password

### PR DESCRIPTION
**Pull request by artemislena**
Trying https://github.com/iv-org/documentation/pull/458 again.

### CODE REVIEW

**Comment by unixfox** [comment]
ping for review @TheFrenchGhosty

File: docs/register-user.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Registering users manually
+You might want to disable registration in your [instance config](/configuration), but still have a quick way to manually register users upon request. To do so, first set up a separate instance that only listens on localhost, has registration enabled,
```
Comment (TheFrenchGhosty) [inline docs/register-user.md:2]: Add a blank line after the header please

File: mkdocs.yml [hunk 2]
```diff
@@ -33,6 +33,7 @@ nav:
     - 'ipv6-rotator.md'
     - 'captcha-bug.md'
     - 'anti-captcha.md'
+    - 'register-user.md'
```
Comment (TheFrenchGhosty) [inline mkdocs.yml:36]: Please don't make a change to this file, I'll do it myself when it's merged

**Review by TheFrenchGhosty — CHANGES_REQUESTED** [review]

File: docs/register-user.md (hunk 1 above)
Comment (artemislena) [reply docs/register-user.md]: T.: Done!

**Comment by unixfox** [comment]
@TheFrenchGhosty please add to the file mkdocs.yaml: https://github.com/iv-org/documentation/pull/459#discussion_r1571079082

**Comment by TheFrenchGhosty** [comment]
@unixfox done https://github.com/iv-org/documentation/commit/377e5d6e741e86f8a64af01eb1bb8b559ca97820

**Pull request merged**